x-pack/package.json

```diff
@@ -100,13 +111,16 @@
     "pdfjs-dist": "^2.0.943",
     "pixelmatch": "4.0.2",
     "proxyquire": "1.7.11",
-    "react-test-renderer": "^16.2.0",
+    "react-docgen-typescript-loader": "^3.0.0",
+    "react-docgen-typescript-webpack-plugin": "^1.1.0",
+    "react-test-renderer": "^16.7.0",
     "redux-test-utils": "0.2.2",
     "rsync": "0.4.0",
     "run-sequence": "^2.2.1",
     "sass-loader": "^7.1.0",
     "simple-git": "1.37.0",
     "sinon": "^5.0.7",
+    "storybook-addon-jsx": "^5.4.0",
```

Comment:
I have pretty much nil experience with storybook.  What's the JSX part supposed to do?  In the UI looks like it's always blank.

Reply:
I can remove this, sure.